server/tokens.go

```diff
@@ -151,7 +211,10 @@ func (c *Server) blindedTokenBulkRedeemHandler(w http.ResponseWriter, r *http.Re
 	}
 
 	for _, token := range request.Tokens {
-		issuer, appErr := c.getLatestIssuer(token.Issuer)
+		// todo: this code seems to be from an old version - we use the `redeemTokenWithDB`, and we have no tests, so I
+		// assume that is no longer used, hence the usage of Cohort 1.
+		issuer, appErr := c.getLatestIssuer(token.Issuer, 1)
+
```

Comment:
@amirsaber can you comment on if this code is used?

Reply:
no it's not being used. Ledger branch uses better way for bulk redemption and would be good to adopt so ads can also bulk redeem